src/libs/bmpv3converter.c

```diff
@@ -0,0 +1,495 @@
+//
```

Comment:
Не нашёл места, где бы использовались битовые операции для конвертации в негатив

Reply:
В функции make_bmpv3_negative используется побитовое отрицание, больше нигде не используются, но я надеялся этого хватит
```suggestion
void make_bmpv3image_negative(BMPV3IMAGE* bmpv3image) {
    if(bmpv3image->color_table == NULL) {
        for(int x = 0; x < bmpv3image->header_meta->width; x++) {
            for(int y = 0; y < bmpv3image->header_meta->height; y++)
                set_pixel(x, y, ~bmpv3image->pixel_array[y * bmpv3image->header_meta->width + x], bmpv3image);
        }
    }
    else {
        for(unsigned int i = 0; i < bmpv3image->header_meta->number_of_colors; i++)
            bmpv3image->color_table[i] = (~bmpv3image->color_table[i]) & 0x00FFFFFF;
    }
}
```